src/app/pages/transaction-request/components/submit-action.tsx

```diff
@@ -25,7 +25,7 @@ export function SubmitAction() {
   const { isLoading } = useLoading(LoadingKeys.SUBMIT_TRANSACTION);
   const error = useTransactionError();
 
-  const isDisabled = !!error || !values.fee;
+  const isDisabled = !!error || !(values.fee >= 0);
```

Comment:
This is an odd way to invert the expression: If the fee is _not_ greater than or equal to 0.

Isn't this the same as `values.fee < 0`

Reply:
Also, `fee` is typed as `number | string`

Reply:
True, did too quick but I think we want to check for an empty string here too.